src/extension.ts

```diff
@@ -822,13 +822,41 @@ export async function activate(context: vscode.ExtensionContext): Promise<any> {
     vscode.commands.registerCommand("vscode-objectscript.pickProcess", async (config) => {
       const system = config.system;
       const api = new AtelierAPI(vscode.window.activeTextEditor?.document.uri);
-      const convert = (data) =>
-        data.result.content.map(
-          (process: AtelierJob): vscode.QuickPickItem => ({
-            label: process.pid.toString(),
-            description: `Namespace: ${process.namespace}, Routine: ${process.routine}`,
-          })
-        );
+      const convert = async (jobData) => {
+        // NOTE: We do not know if the current user has permissions to other namespaces, so lets only fetch the job infos
+        // for the current namespace.
+        const currNamespaceJobs: { [k: string]: string } = await api
+          .actionQuery("CALL Ens.Job_Enumerate()", [])
```

Comment:
What happens if the namespace isn't an Ensemble / Interoperability-enabled one?

Reply:
The actionQuery will return an error saying that no such function exists. Tried searching a way to check if procedure exists and resolve the issue in sql but couldn't find anything so I added a check to the error handling code to ignore this error.

Reply:
I have suggested a less strict text check in case error message gets localised.